rpctest/README.md

```diff
@@ -4,13 +4,23 @@ rpctest
 [![Build Status](https://travis-ci.org/btcsuite/btcd.png?branch=master)]
 (https://travis-ci.org/btcsuite/btcd)
 
-Package rpctests provides a btcd-specific RPC testing harness crafting and executing integration tests by driving a `btcd` instance via the `RPC` interface. Each instance of a active harness comes equipped with a simple in-memory HD wallet capable of properly syncing to the generated chain, creating new addresses, and crafting fully signed transactions paying to an arbitrary set of outputs. 
-
-This package was designed specifically to act as an RPC testing harness for `btcd`. However, the constructs presented are general enough to be adapted to any project wishing to programmatically drive a `btcd` instance of its systems/integration tests. 
+Package rpctests provides a btcd-specific RPC testing harness crafting and
+executing integration tests by driving a `btcd` instance via the `RPC`
+interface. Each instance of an active harness comes equipped with a simple
+in-memory HD wallet capable of properly syncing to the generated chain,
+creating new addresses, and crafting fully signed transactions paying to an
+arbitrary set of outputs. 
+
+This package was designed specifically to act as an RPC testing harness for
+`btcd`. However, the constructs presented are general enough to be adapted to
+any project wishing to programmatically drive a `btcd` instance of its
+systems/integration tests. 
 
 ## Documentation
 
-[![GoDoc](https://godoc.org/github.com/btcsuite/btcd/rpctest?status.png)]
+[![ISC License]

```

Comment:
Please see the [blockchain README.md](https://github.com/btcsuite/btcd/blob/763f731c5c97ab2f659f0c7dd3e6a8f48c723daa/blockchain/README.md) for example.  Notice how the entire `## Documentation` section is gone and these are at the top along with the build status.